Functions/BouncingNumbers.cs

```diff
@@ -5,10 +5,12 @@ public class BouncingNumbers
     public static bool IsAcending(int num)
     {
         int digit;
```

Comment:
I understand what you did here, but these variables don't need to retain their value. Please move these lines into the loop.

Reply:
Declaring a variable is not time consuming, but I understand you concern.
About the error, isn't it because you didn't remove the deceleration from before?